src/MobileDelivery/MobileConfirm.js

```diff
@@ -3,50 +3,36 @@ import moment from 'moment';
 import './Mobile.css';
 
 class MobileConfirm extends React.Component {
-    onSubmit(e){
-        e.preventDefault();
-        var data = {
-            raSignature: e.target.signature.value,
-            raPrintName: e.target.print.value,
-            timeCompleted: moment().format()
-        };
-        this.props.nextStep();
-        this.props.saveValues(data);
-    }
-
-    componentDidMount(){
-    }
-
     render() {
         return (
             <div className="mobile-confirmation-container">
                 <div className="mobile-confirm-card">
                     <div className="mobile-card-line"></div>
                     <p className="ms-header">Completion Summary</p>
-                    <p id="confirm-time">{moment(this.props.currentDelivery.timeDelivered).calendar()}</p>
+                    <p id="confirm-time">{moment.unix(this.props.deliveryObj.deliveredInfo.timestamp).calendar()}</p>
```

Comment:
I think this line is actually supposed to be something like "Thursday 1/13/2018" (according to Figma)